packages/chain-mon/.prettierrc.js

```diff
@@ -1,3 +1,3 @@
 module.exports = {
-  ...require('../../.prettierrc.js'),
+  ...require('../@eth-optimism/configs/.prettierrc.js'),
```

Comment:
```suggestion
  ...require('@eth-optimism/configs/.prettierrc.js'),
```
These relative imports all need to be fixed